src/index.jsx

```diff
@@ -46,4 +52,8 @@ const AppWrapper = () => (
   </Provider>
 );
 
-ReactDOM.render(<AppWrapper />, document.getElementById('root'));
+if (routeConfig.isCurrentRoutePublic() || apiClient.isAuthenticated()) {
```

Comment:
Can we add a comment here indicating this is outside the rendering of react in order to short-circuit loading the app just to redirect.